libs/ui/elements/src/lib/metadata-contact/metadata-contact.component.ts

```diff
@@ -34,6 +34,37 @@ export class MetadataContactComponent {
     )
   }
 
+  parseAddress(inputAddress) {
+    const addressParts = inputAddress.split(',').map((part) => part.trim())
```

Comment:
The only thing that worries me is if one address doesn't have any commas (as we spoke about). Is this a case that can happen? @fgravin

Reply:
yes definitely, there's no rule for addresses as far as I know; but then it'll simply end up being a long line, so we just need to make sure that the text overflow works well :) (looks to be the case in my tests)

Reply:
Okay in this case I will keep it simple as you were proposing @jahow and only split by ',' and if that's not there, it will be a long line of text which overflows, as you said. I implemented it to fit the design, but I guess there are too many different variations of the user input.